firedrake/slope_limiter/limiter.py

```diff
@@ -0,0 +1,38 @@
+from abc import ABCMeta, abstractmethod
+
+
+class Limiter(object):
+    __metaclass__ = ABCMeta
+
+    @abstractmethod
+    def __init__(self, space):
+        """
+        Abstract Limiter class for all limiters to implement its methods.
+
+        :param space: FunctionSpace instance
+        """
+        pass
+
+    @abstractmethod
+    def apply(self, field):
+        """
```

Comment:
I'd prefer making these one line, rather than three lines. That is, if the docstring is only one line long, I put the quotes on the same line.